pkg/util/certificates/certificates.go

```diff
@@ -0,0 +1,103 @@
+package certificates
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"math/big"
+	"net"
+	"os"
+	"time"
+)
+
+func CreateCertificates(caCertPath, caKeyPath, serverCertPath, serverKeyPath string) error {
```

Comment:
It might be worth breaking this down into two seperate functions one for the ca cert/key and one for the server.

Reply:
I would also amend the name to clarify that this produces *test* certificates and shouldn't be (accidentally) used to generate real ones.

Reply:
On a second look, perhaps they might also be better in pkg/test/utils/certificates rather than pkg/util?